homeassistant/components/matter/event.py

```diff
@@ -104,9 +104,7 @@ def _update_from_device(self) -> None:
         """Call when Node attribute(s) changed."""
 
     @callback
-    def _on_matter_node_event(
-        self, event: EventType, data: MatterNodeEvent
-    ) -> None:  # noqa: F821
+    def _on_matter_node_event(self, event: EventType, data: MatterNodeEvent) -> None:  # noqa: F821
```

Comment:
This seems like an odd change. Instead of reformatting, the noqa command should have been moved.